2024-01-17 15:13:46,124:DEBUG:certbot._internal.main:certbot version: 2.8.0
2024-01-17 15:13:46,124:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2024-01-17 15:13:46,124:DEBUG:certbot._internal.main:Arguments: ['--webroot', '-w', '/var/www/certbot', '--email', 'tjfgkssk9472@gmail.com', '--agree-tos', '--no-eff-email', '-d', 'tairot.online']
2024-01-17 15:13:46,125:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2024-01-17 15:13:46,302:DEBUG:certbot._internal.log:Root logging level set at 30
2024-01-17 15:13:46,308:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2024-01-17 15:13:46,309:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7ff9c9924a30>
Prep: True
2024-01-17 15:13:46,309:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7ff9c9924a30> and installer None
2024-01-17 15:13:46,310:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2024-01-17 15:13:46,373:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/1519022676', new_authzr_uri=None, terms_of_service=None), 053f50e92556dc2ae6fb1958ad04b4cb, Meta(creation_dt=datetime.datetime(2024, 1, 16, 8, 39, 49, tzinfo=<UTC>), creation_host='1b0b13fa81e7', register_to_eff=None))>
2024-01-17 15:13:46,374:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2024-01-17 15:13:46,377:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2024-01-17 15:13:46,876:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 752
2024-01-17 15:13:46,877:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 15:13:46 GMT
Content-Type: application/json
Content-Length: 752
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "KhbW5rSxq0s": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417",
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.3-September-21-2022.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-02/renewalInfo/",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert"
}
2024-01-17 15:13:46,880:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for tairot.online
2024-01-17 15:13:46,886:DEBUG:acme.client:Requesting fresh nonce
2024-01-17 15:13:46,887:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2024-01-17 15:13:47,048:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2024-01-17 15:13:47,049:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 15:13:46 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 70SDU6HFM40L5yCSc6l45BbXqLDm6Xzr-9JT_1wTKmVlXTLG7yo
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2024-01-17 15:13:47,050:DEBUG:acme.client:Storing nonce: 70SDU6HFM40L5yCSc6l45BbXqLDm6Xzr-9JT_1wTKmVlXTLG7yo
2024-01-17 15:13:47,050:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "tairot.online"\n    }\n  ]\n}'
2024-01-17 15:13:47,052:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICI3MFNEVTZIRk00MEw1eUNTYzZsNDVCYlhxTERtNlh6ci05SlRfMXdUS21WbFhUTEc3eW8iLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "GA3QYAuetecKAOZtJz-MIOq1XetpnGamS1dq7ItbpmqR_-L7eNsrjQZB6q0p6W7uzWYgzlGBmOt5EHSkHH2ej1vC7wBGQIGoJgWn_Ly92TM0hsBiANKeRT24mZalrvsMBC09P5HxGRUVpWZ3uI7olKmRFjGAXhwUnFBwnNPoK1LH-4hm9UePf5lM5wEJMpf-rvh_jL0vaXAKZ1IcjdU5a2MCoChenZ0W0lyDiFPmPLKiWVntmRinDqSKdl4q5anrMF9MsZZXcWaReFsexdJsmcMCNOiXyRbaa4ZFTkVwi3MlWcvwJw0M_aia0PMaJs80gWChjm7ne8TqimV3AQkiOQ",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogInRhaXJvdC5vbmxpbmUiCiAgICB9CiAgXQp9"
}
2024-01-17 15:13:47,247:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 339
2024-01-17 15:13:47,248:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Wed, 17 Jan 2024 15:13:46 GMT
Content-Type: application/json
Content-Length: 339
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/1519022676/237380886306
Replay-Nonce: 70SDU6HFdAytIUPySHwkY8V3e1tUpWGkW_vnYJWcyaWAjoNKE74
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2024-01-24T15:13:46Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "tairot.online"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz-v3/305181032156"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/1519022676/237380886306"
}
2024-01-17 15:13:47,249:DEBUG:acme.client:Storing nonce: 70SDU6HFdAytIUPySHwkY8V3e1tUpWGkW_vnYJWcyaWAjoNKE74
2024-01-17 15:13:47,249:DEBUG:acme.client:JWS payload:
b''
2024-01-17 15:13:47,251:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/305181032156:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICI3MFNEVTZIRmRBeXRJVVB5U0h3a1k4VjNlMXRVcFdHa1dfdm5ZSldjeWFXQWpvTktFNzQiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMwNTE4MTAzMjE1NiJ9",
  "signature": "qgAcKC_alEsF5G6j1zTTopeNjbT_wxawqSejjLow27skDg8Gc-QtVhkwz9Ha-LRwIMWieeKDWekQaVDdL-hVXlg2Igyy1hWTRO8whh-3d9rFd-wSdRv6hsWRkbl5pZD_UJSYKczyd_7-gc8XiO1jYoHtRJ_wC4cpVr-JkyPZLttwIHc7OkK3jcORAKFyYiY1zB3M1WR3yecswtIO8OYZ3h6p54SuFCFx_QemuhIyhNL0-zQBoSNqtM3cRIRR4UpSHK5PmHkX4uAqORLchPftvVpqXLqnQizXoEbjcWAsAug28TGv2Rx72AWFd2gKsNJAQo2aTgBdx0UVw1Me-D9gjg",
  "payload": ""
}
2024-01-17 15:13:47,409:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/305181032156 HTTP/1.1" 200 797
2024-01-17 15:13:47,410:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 15:13:46 GMT
Content-Type: application/json
Content-Length: 797
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 70SDU6HFeWABImcoi-tyGyiDKClxY1RqoCRkzpHsupWJWW3pVK4
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "tairot.online"
  },
  "status": "pending",
  "expires": "2024-01-24T15:13:46Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305181032156/3GcAOQ",
      "token": "-mmdkrmDeNJbcI-UXUh5xT0AacfxdssMGo2gVDLUTfg"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305181032156/2x3dDA",
      "token": "-mmdkrmDeNJbcI-UXUh5xT0AacfxdssMGo2gVDLUTfg"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305181032156/R0lDYA",
      "token": "-mmdkrmDeNJbcI-UXUh5xT0AacfxdssMGo2gVDLUTfg"
    }
  ]
}
2024-01-17 15:13:47,410:DEBUG:acme.client:Storing nonce: 70SDU6HFeWABImcoi-tyGyiDKClxY1RqoCRkzpHsupWJWW3pVK4
2024-01-17 15:13:47,411:INFO:certbot._internal.auth_handler:Performing the following challenges:
2024-01-17 15:13:47,412:INFO:certbot._internal.auth_handler:http-01 challenge for tairot.online
2024-01-17 15:13:47,412:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2024-01-17 15:13:47,413:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2024-01-17 15:13:47,420:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/-mmdkrmDeNJbcI-UXUh5xT0AacfxdssMGo2gVDLUTfg
2024-01-17 15:13:47,423:DEBUG:acme.client:JWS payload:
b'{}'
2024-01-17 15:13:47,424:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall-v3/305181032156/3GcAOQ:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICI3MFNEVTZIRmVXQUJJbWNvaS10eUd5aURLQ2x4WTFScW9DUmt6cEhzdXBXSldXM3BWSzQiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLXYzLzMwNTE4MTAzMjE1Ni8zR2NBT1EifQ",
  "signature": "bMCzbOt95b6SfXuQQSR3tt5Qe_AYKUa-axlHr_iMTNjiGv9wJLCJj6QgWB1FinJm4ICSWDBMnfCTodYoeJnguqu6c250Nur4VQmfu-lBW3NmXAqDTXR22Pi4ZKDvSgyn0bdtZxUTCvEdHht8d_a21Mj-Bh-gerLUfVExB5Ij74DueK8B6nECjrgSU_LNwFtOSoRs56Vh1VhW-YTmswLN5IauEef8kwtD1ccsSTdXl0Y9z2DGWU6a3hn1hnRzHXVclub7p9xkafI7oVdqSDny4Sqxfszb5hl5jylsAXU6PMVeOpj5z8oSPiBja6Dx2mbFfw-MsGUnz1oDvLhvnBD9dQ",
  "payload": "e30"
}
2024-01-17 15:13:47,587:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/305181032156/3GcAOQ HTTP/1.1" 200 187
2024-01-17 15:13:47,588:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 15:13:47 GMT
Content-Type: application/json
Content-Length: 187
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz-v3/305181032156>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall-v3/305181032156/3GcAOQ
Replay-Nonce: LPSUY_lxWl8g2A2rjOrDqo2hLSt2909aDcKBmIkTDO8v3GocXBE
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "status": "pending",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305181032156/3GcAOQ",
  "token": "-mmdkrmDeNJbcI-UXUh5xT0AacfxdssMGo2gVDLUTfg"
}
2024-01-17 15:13:47,588:DEBUG:acme.client:Storing nonce: LPSUY_lxWl8g2A2rjOrDqo2hLSt2909aDcKBmIkTDO8v3GocXBE
2024-01-17 15:13:47,589:INFO:certbot._internal.auth_handler:Waiting for verification...
2024-01-17 15:13:48,590:DEBUG:acme.client:JWS payload:
b''
2024-01-17 15:13:48,592:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/305181032156:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICJMUFNVWV9seFdsOGcyQTJyak9yRHFvMmhMU3QyOTA5YURjS0JtSWtURE84djNHb2NYQkUiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMwNTE4MTAzMjE1NiJ9",
  "signature": "LkL8tpc7D7h2nvPbKnfoAyNyvJ7IG7tPAqajLWg2O3FDSa2yqWTnEnQ3Wt5Fh1Y4kANcFzYgpjlLnfZMXDE6K0bfuYSQT8QlSD-iZRPQhtaChb8VVXkeq_pVQRdA2VeYodCziT8XJviZnL_2Clg_Is5UlEuUPCosWg21Oop6DQE9uyrMQxrdTLqwy4WE3ggqlPI0OGW_a3_vitt6p34FpvTpLTTW7LRgYoOIZ2SVyxawTvrXx82BypC6JA3F648chypDduLXnZEhHht0pzE5cElJMWypSZjPe_SbA_ANspAg1vrM-6NG_Ao7CMB8NmqIMlW_DyBl4XJB1o_wC-dS2g",
  "payload": ""
}
2024-01-17 15:13:48,781:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/305181032156 HTTP/1.1" 200 1023
2024-01-17 15:13:48,782:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 15:13:48 GMT
Content-Type: application/json
Content-Length: 1023
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 70SDU6HFMSI3nDrpcM-hzVdUbIWwTiDjwl_6BjslXIcqrOj-EYs
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "tairot.online"
  },
  "status": "invalid",
  "expires": "2024-01-24T15:13:46Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "invalid",
      "error": {
        "type": "urn:ietf:params:acme:error:unauthorized",
        "detail": "43.202.208.226: Invalid response from http://tairot.online/.well-known/acme-challenge/-mmdkrmDeNJbcI-UXUh5xT0AacfxdssMGo2gVDLUTfg: 404",
        "status": 403
      },
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305181032156/3GcAOQ",
      "token": "-mmdkrmDeNJbcI-UXUh5xT0AacfxdssMGo2gVDLUTfg",
      "validationRecord": [
        {
          "url": "http://tairot.online/.well-known/acme-challenge/-mmdkrmDeNJbcI-UXUh5xT0AacfxdssMGo2gVDLUTfg",
          "hostname": "tairot.online",
          "port": "80",
          "addressesResolved": [
            "43.202.208.226"
          ],
          "addressUsed": "43.202.208.226"
        }
      ],
      "validated": "2024-01-17T15:13:47Z"
    }
  ]
}
2024-01-17 15:13:48,782:DEBUG:acme.client:Storing nonce: 70SDU6HFMSI3nDrpcM-hzVdUbIWwTiDjwl_6BjslXIcqrOj-EYs
2024-01-17 15:13:48,783:INFO:certbot._internal.auth_handler:Challenge failed for domain tairot.online
2024-01-17 15:13:48,783:INFO:certbot._internal.auth_handler:http-01 challenge for tairot.online
2024-01-17 15:13:48,784:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: tairot.online
  Type:   unauthorized
  Detail: 43.202.208.226: Invalid response from http://tairot.online/.well-known/acme-challenge/-mmdkrmDeNJbcI-UXUh5xT0AacfxdssMGo2gVDLUTfg: 404

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2024-01-17 15:13:48,785:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2024-01-17 15:13:48,785:DEBUG:certbot._internal.error_handler:Calling registered functions
2024-01-17 15:13:48,786:INFO:certbot._internal.auth_handler:Cleaning up challenges
2024-01-17 15:13:48,786:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/-mmdkrmDeNJbcI-UXUh5xT0AacfxdssMGo2gVDLUTfg
2024-01-17 15:13:48,790:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2024-01-17 15:13:48,790:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1869, in main
    return config.func(config, plugins)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 143, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2024-01-17 15:13:48,791:ERROR:certbot._internal.log:Some challenges have failed.
